sdk/core/azure-core/src/main/java/com/azure/core/http/HttpRequest.java

```diff
@@ -158,6 +174,15 @@ public HttpRequest setHeader(String name, String value) {
      * @return the content to be send
      */
     public Flux<ByteBuffer> getBody() {
+        return body == null ? null : body.toFluxByteBuffer();
+    }
+
+    /**
+     * Get the request content.
+     *
+     * @return the content to be send
```

Comment:
Grammatical mistake copied from the other API

```suggestion
     * @return the content to be sent
```